plugins/catalog/report.api.md

```diff
@@ -154,35 +154,7 @@ export interface CatalogSearchResultListItemProps {
 }
 
 // @public
-export const CatalogTable: {
-  (props: CatalogTableProps): JSX_2.Element;
-  columns: Readonly<{
```

Comment:
We can't make this change just yet, it's a breaking change. 

If we want to go down the hooks route, it's most likely what we want to do is move the hook to `-react` package, and then deprecate this export. But with the upcoming changes to NFS, that could also have an affect on how these columns are structured and used.

I'm wondering if we should just skip this refactor for now, and revisit once we have a clearer understanding of what we need to do?

Reply:
Ok, I will try some effort to stop this breaking change.